hs_core/decorators.py

```diff
@@ -0,0 +1,24 @@
+from functools import wraps
+from django.shortcuts import redirect
+from django.utils.decorators import method_decorator
+
+
+def profile_required(view_func):
+    '''Checks all required UserProfile fields.'''
```

Comment:
I'd fill out the docstring on this more - say what is supposed to happen when an incomplete profile is encountered. Say whay happens when a complete one is encountered.

Reply:
Also say that it is intended to be used only on methods serving requests.